dir.props

```diff
@@ -187,7 +187,7 @@
     <NuGetRuntimeIdentifier Condition="'$(TargetGroup)' == 'netcore50aot'">win8-aot</NuGetRuntimeIdentifier>
 
     <!-- workaround Dev14 issue with nuget targets -->
-    <RuntimeIndentifier>$(NuGetRuntimeIdentifier)</RuntimeIndentifier>
+    <RuntimeIdentifier>$(NuGetRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
This misspelling is intentional.  The Dev14 RTM targets had the same misspelling.